pom.client.xml

```diff
@@ -972,6 +972,150 @@
          </plugins>
       </build>
     </profile>
+
+    <profile>
+      <!-- separate profile for windows as the executable on windows is named npx.cmd and npx on other os families -->
+      <id>readme-codesnippet-windows</id>
+      <activation>
+        <property>
+          <name>readme-codesnippet-windows</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <version>1.2.1</version> <!-- {x-version-update;org.codehaus.mojo:exec-maven-plugin;external_dependency} -->
+            <executions>
+              <execution>
+                <id>code-snippet-for-readme-windows</id>
+                <phase>prepare-package</phase>
+                <goals>
+                  <goal>exec</goal>
+                </goals>
+                <configuration>
+                  <executable>npx.cmd</executable>
+                  <arguments>
+                    <argument>embedme</argument>
+                    <argument>sdk/*/azure-*/README.md</argument>
+                  </arguments>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <!-- Profiles for updating README file using embedme tool to inject code snippets from compilable Java source files.
+     Activate the profile below to update README files when corresponding Java files are updated. -->
+    <profile>
+      <id>readme-codesnippet</id>
+      <activation>
+        <property>
+          <name>readme-codesnippet</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <version>1.2.1</version> <!-- {x-version-update;org.codehaus.mojo:exec-maven-plugin;external_dependency} -->
+            <executions>
+              <execution>
+                <id>code-snippet-for-readme</id>
+                <phase>prepare-package</phase>
+                <goals>
+                  <goal>exec</goal>
+                </goals>
+                <configuration>
+                  <executable>npx</executable>
+                  <arguments>
+                    <argument>embedme</argument>
+                    <argument>sdk/*/azure-*/README.md</argument>
+                  </arguments>
```

Comment:
Would  this also cover the README in modules. For example 

sdk/eventhubs/README.md
sdk/eventhubs/azure-messaging-eventhubs/README.md

Reply:
As discussed, the code samples should only be at the module level and shouldn't be at sdk/*/ level. So, not including those READMEs here.